packages/api/endpoints/granules.js

```diff
@@ -212,11 +212,23 @@ async function bulk(req, res) {
     tableName: process.env.AsyncOperationsTable
   });
 
+  let description;
+
+  if (payload.query) {
+    description = `Bulk run ${payload.workflowName} on ${payload.query.size} granules`;
+  } else if (payload.ids) {
+    description = `Bulk run ${payload.workflowName} on ${payload.ids.length} granules`;
+  } else {
+    descritpion = `Bulk run on ${payload.workflowName}`;
```

Comment:
description is spelled incorrectly. Should have tests to cover all of these description scenarios.

Reply:
Fixed the typo and added tests for the description